claim/netdata-claim.sh.in

```diff
@@ -131,6 +146,7 @@ do
                 -insecure) INSECURE=1 ;;
                 -proxy=*) PROXY=${arg:7} ;;
                 -noproxy) NOPROXY=yes ;;
+                -user=*) NETDATA_USER=${arg:6} ;;
```

Comment:
This code will never be reached if "This script must be run by the $NETDATA_USER user account" has already been returned.

Reply:
Fixed.